### ISSUE TITLE
Homepage jumbotron is hard to read

### ISSUE THREAD

**Issue by dhornbein** [dhornbein/denverfreelancing.com#15]
The image contains a lot of white and the white text clashes and is hard to read. The image whites could be reduced or text shadow could be applied to the text.

![screen shot 2019-02-27 at 9 27 33 pm](https://user-images.githubusercontent.com/358427/53541342-a3fa1a00-3ad6-11e9-8006-7776cb2f91fd.png)

**Comment by tomwhartung** [comment]
I got into the gimp and posterized the original image, and converted it to black-and-white.  Being unsure whether we want to stick with this image, I didn't spend much time on it.

"Do we want to use this image?" would be my first question.

If so, I could get back into the gimp and work on addressing this issue, else I guess that would be another issue, which would make this one moot.

**Comment by tomwhartung** [comment]
Having just noticed the "good first issue" tag, if someone else wants to take care of this, that's fine!

**Comment by dhornbein** [comment]
This would be a good issue to hand off to a designer.

**Comment by tomwhartung** [comment]
Yes!  Here is the link for the group photos, so they can start from scratch with a different photo if they want:

https://drive.google.com/drive/folders/1c907O0EON8ZX5ywbNuK1jvKsT5MB_KW4

The one I used is at the very end.
